src/Contract Test Generator.postman_collection.json

```diff
@@ -1520,31 +1547,5 @@
 				]
 			}
 		}
-	],
-	"variable": [
-		{
-			"key": "coll-apiIds",
-			"value": ""
-		},
-		{
-			"key": "coll-apiId",
-			"value": ""
-		},
-		{
-			"key": "coll-versionId",
-			"value": ""
-		},
-		{
-			"key": "coll-schemaId",
-			"value": ""
-		},
-		{
-			"key": "coll-schema",
-			"value": ""
-		},
-		{
-			"key": "coll-schemaTests",
-			"value": ""
-		}
```

Comment:
When exporting, it wasn't there, I don't think it's necessary to make the collection work